sphinx/texinputs/footnotehyper-sphinx.sty

```diff
@@ -174,6 +174,9 @@
      \unrestored@protected@xdef\@thefnmark{\thempfn}%
     \endgroup
     \@footnotemark
+% Sphinx addition: store internal hyperref anchor reference to allow
+% multiple clickable reference to same numbered footnote
+    \expandafter\xdef\csname FNH@!#1\endcsname{\Hy@footnote@currentHref}%
```

Comment:
This kind of hack may in future become a feature of upstream footnotehyper LaTeX package

Reply:
Here the #1 is anyhow required by the LaTeX core syntax to be something TeX recognizes as number. But here we use it into `\csname`, so it can not be a TeX count register. If we wanted maximum flexibility (as will probably be done upstream at footnotehyper once they get jealous of our work here), we would use `\the\numexpr#1\relax` here. But anyhow Sphinx mark-up uses exclusively explicit digits this is not needed.